app/libs/utils.js

```diff
@@ -14,3 +14,15 @@ export function timeToString(time) {
             + seconds;
     return r;
 };
+
+export function hasLocalStorage() {
```

Comment:
I think you can safely ommit this test feature, as czy target browsers (last versions only) supports all `localStorage`: http://caniuse.com/#search=localstorage (expect OperaMini which, in fact, isn't a browser and can't be used to access Cozy).